packages/swirl-components/src/components/swirl-banner/swirl-banner.tsx

```diff
@@ -108,19 +113,19 @@ export class SwirlBanner {
           role={this.importance}
           tabIndex={0}
         >
-          {showIcon && (
-            <span
-              aria-hidden="true"
-              class="banner__icon"
-              innerHTML={icon}
-              ref={(el) => (this.iconEl = el)}
-            ></span>
-          )}
-          <span class="banner__content" id="content" part="banner__content">
-            {this.content}
-          </span>
+          <div class="banner__content" id="content" part="banner__content">
+            {showIcon && (
+              <span
+                aria-hidden="true"
+                class="banner__icon"
+                innerHTML={icon}
+                ref={(el) => (this.iconEl = el)}
+              ></span>
+            )}
+            <span class="banner__text">{this.content}</span>
+          </div>
           {showControls && (
-            <span class="banner__controls">
+            <div class="banner__controls">
```

Comment:
Not sure, if this was caused by your changes, but the controls could use some spacing on the left.
<img width="517" alt="Bildschirmfoto 2024-05-14 um 13 42 26" src="https://github.com/getflip/swirl/assets/1019414/578d0a4c-f4c1-4cf1-8b78-51d0fe0864b5">

Reply:
oops i forgot to readd the gap property. will change